featurebyte/session/base.py

```diff
@@ -535,6 +535,15 @@ async def execute_query(
         """
         try:
             bytestream = self.get_async_query_stream(query=query, timeout=timeout)
```

Comment:
Since `get_async_query_stream` return an async generator, the exception is raised only at line 540. To fix the issue, we move the logic into the try-except block.